doc/develop/microscope.rst

```diff
@@ -89,7 +90,57 @@ Detectors:
    only in confocal microscopes (photo-detector0, photo-detectorN).
 
 System:
- * lens: Contains parameters concerning the parabolic mirror and the lens system.
+ * lens: Defines the optical parameters (e.g magnification) of the optical path.
+
+Roles of components found in the CryoSECOM (ENZEL) system:
+----------------------------------------------------------
+The role of the microscope is cryo-secom.
+
+Emitters:
+ * e-beam: Electron beam of the SEM to scan the sample.
+ * light: Controls the excitation light of the fluorescence microscope.
+
+Actuators:
+ * ebeam-focus: Changes the focus position of the e-beam. It has one axis: z. 
+ * filter: Emission filter to select a specific wavelength band.
+ * stigmator: Controls the rotation of the astigmatic lens. It has one axis rz. It has one metadata:
+
+    1. POS_COR: Position correction to reduce astigmatism.
+
+ * stage: Moves the sample. It has 3 linear axes (x, y, z) and 2 rotational axes (rx, rz). Stage component has the following metadata:
+  
+    1. FAV_POS_DEACTIVE: Loading/unloading position.
+    2. FAV_POS_ACTIVE: Imaging position.
+    3. FAV_POS_COATING: Coating position of the gas injection system (GIS). 
+    4. POS_ACTIVE_RANGE: The allowed position range during the FM/SEM imaging.  
+    5. OVERVIEW_RANGE: The position range for the overview acquisition. 
+    6. ION_BEAM_TO_SAMPLE_ANGLE: Angle of the e-beam with the sample when rx = 0 
+   
+ * focus: Changes the distance between the sample and the optical detectors. It has one axis: z. It has three metadata:
+  
+    1. FAV_POS_DEACTIVE: The safe focus position along z-axis.   
+    2. FAV_POS_ACTIVE: The good focus position 
+    3. POS_COR: Position correction.
+   
+ * align: Alignment actuator. It has 2 axes: x and y. It has also 2 metadata:
+  
+    1. FAV_POS_ACTIVE: The position corresponding to alignment. 
+    2. FAV_POS_DEACTIVE: The safe position to go such that the stage cannot hit the objective lens 
+
+Detectors:
+ * se-detector: Secondary electron detector of the SEM. 
+ * ccd: The main optical pixelated detector. It includes 1 metadata:
+
+    1. FAV_POS_ACTIVE: Focus position at initialisation.
+
+System:
+ * sample-thermostat: Controls the temperature of the sample. The metadata are:
```

Comment:
-> "Controls the temperature of the sample *finely*."

To clarify: typically the cooler cools down to a very low temperature (fixed by some physical limitations), and the sample-thermostat heats up the sample to exactly get the requested temperature.